Build.cmake

```diff
@@ -85,6 +85,10 @@ ENDIF()
    ELSE()
          list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/crashhandler_windows.cpp ${LOG_SRC}/g3log/stacktrace_windows.hpp ${LOG_SRC}/stacktrace_windows.cpp)
    ENDIF (MSVC OR MINGW)
+   
+   IF (MSVC) 
+         SET(CMAKE_DEBUG_POSTFIX "_d")
```

Comment:
I was hesitant to add this but it seems that the overwriting of library files (debug/release) is a common issue for Windows. 

I've been away (injured) from coding from some time so I'm just getting back to this now.  I still have to verify that this works in conjunction with g3sinks PR https://github.com/KjellKod/g3sinks/pull/20